packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -922,6 +922,15 @@ export default {
                     this.shouldShowSpinner = true;
                 }
             }, this.spinnerTimeout);
+        },
+
+        getReferralState () {
+            const cookieName = `je-rw-menu-referral-state-${this.restaurant.id}`;
+            const referralCookie = this.$cookies.get(cookieName);
+
+            return referralCookie && referralCookie.menuReferralState
+                ? 'ReferredByWeb'
```

Comment:
Should this be hardcoded to 'ReferredByWeb' or should it rather take whatever value is in referralCookie.menuReferralState

Reply:
`referralCookie.menuReferralState` stores either `1` or `0`, so yeah, we need to hardcoded it here 👍